.github/workflows/code_format_checks.yml

```diff
@@ -0,0 +1,23 @@
+# We need to run the format checks on a Linux machine because gofmt
+# asserts platform-specific line endings (see #259)
+name: Code Format Checks
+on: [push, pull_request]
+jobs:
+  code_format_checks:
+    strategy:
+      matrix:
+        go: ['1.16']
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
lgtm, I was reading the logs and there's this job which is already doing the same I think `pull-kubernetes-csi-csi-proxy`, I didn't check the full log though

Reply:
I think you're right. Should we just delete this then?

Reply:
yes please, removing `make all` from the windows job is great though